components/server/src/server.rs

```diff
@@ -1015,6 +1024,65 @@ impl<ER: RaftEngine> TiKVServer<ER> {
             });
     }
 
+    fn init_storage_stats_task(&self, engines: Engines<RocksEngine, ER>) {
+        let config_disk_capacity: u64 = self.config.raft_store.capacity.0;
+        let store_path = self.store_path.clone();
+        let snap_mgr = self.snap_mgr.clone().unwrap();
+        //TODO wal size ignore?
+        self.background_worker
+            .spawn_interval_task(DEFAULT_STORAGE_STATS_INTERVAL, move || {
+                if disk::get_disk_reserved() == 0 {
+                    return;
+                }
```

Comment:
How about checking this before starting the worker, so if the reserve size is zero we don't need to start the worker.

Reply:
but, if users set it back to not zero, then it will not restast.
place here will let users set it online.

Reply:
For this feature, a PR will be specially opened to handle the place holder file. The desired effect is to allow users to dynamically set the reserved size. So the worker is retained.

Reply:
Get it

Reply:
When using tiup to change the configuration parameters, the load operation will restart one node after another. Therefore, this PR does not consider online matters for the time being. 
Done